sdk/search/perf-tests/search-documents/test/getDocumentsCount.spec.ts

```diff
@@ -0,0 +1,252 @@
+import {
+  PerfStressOptionDictionary,
+  PerfStressTest,
+  getEnvVar
+} from "@azure/test-utils-perfstress";
+import {
+  SearchClient,
+  AzureKeyCredential,
+  CountDocumentsOptions,
+  SearchIndexClient,
+  SearchIndex,
+  KnownAnalyzerNames
+} from "@azure/search-documents";
+import { Hotel } from "./hotel";
+
+export class GetDocumentsCountTest extends PerfStressTest<CountDocumentsOptions> {
+  public options: PerfStressOptionDictionary<CountDocumentsOptions> = {
+  };
```

Comment:
"options" here is meant for the test itself.. the framework doesn't use them.
The options that are defined here can be used by the test.. https://github.com/Azure/azure-sdk-for-js/wiki/Writing-Performance-Tests#custom-options

If you don't want to pass any options such as "count of the documents", "size of the documents".. you can simply extend the PerfStressTest without options.
> extends PerfStressTest<CountDocumentsOptions>

to 

> extends PerfStressTest

Reply:
@HarshaNalluru I tried that as the following:

```
export class GetDocumentsCountTest extends PerfStressTest {}
```

It gives me the error:

```
Non-abstract class 'GetDocumentsCountTest' does not implement inherited abstract member 'options' from class 'PerfStressTest<{}>'
```

Is this the correct way?

Reply:
@HarshaNalluru  I have removed the options from the extends part. But, still kept the empty options declaration. Can you please review it now?

Reply:
looking good